diff --git a/src/libs/expo-file-system.ts b/src/libs/expo-file-system.ts
index 313d63e55d3d704c5e4ba6749d97ce336ef9c2e6..80559671abfb0dc65fcc89fa9f32b6e547d1f0da 100644
--- a/src/libs/expo-file-system.ts
+++ b/src/libs/expo-file-system.ts
@@ -57,7 +57,7 @@ type FileSystem = {
 let ExpoFileSystem: FileSystem | undefined;
 
 try {
-  ExpoFileSystem = require('expo-file-system');
+  ExpoFileSystem = require('expo-file-system/legacy');
 } catch {}
 
 export default ExpoFileSystem;
